packages/core/src/TestableMutant.ts

```diff
@@ -12,19 +12,39 @@ export enum TestSelectionResult {
   Success
 }
 
+class TestFilter {
+  public timeSpentScopedTests = 0;
+  public runAllTests = false;
+  public selectedTests: TestSelection[] = [];
+
+  public selectAllTests(runResult: RunResult) {
+    runResult.tests.forEach(testResult => (this.timeSpentScopedTests += testResult.timeSpentMs));
```

Comment:
Could you use the `reduce` function here to make the assignment extra clear? (With perhaps a default value of 0 in the reduce function)